### DISCUSSION TITLE
My ATOMSTACK is lacking precision

### DISCUSSION THREAD

**Discussion by Momboz52** [arkypita/LaserGRBL#1942] [General]
My device seems not to cut the correct path.
As an example you can see in the annexed pictures what I want .[
![image](https://user-images.githubusercontent.com/10050164/199024264-bb059c37-1643-4687-9822-fc817d233a8c.png)
](url)
and what I get [
![image](https://user-images.githubusercontent.com/10050164/199025013-1a270223-fd37-469f-a003-2d13c034e55c.png)
](url)
in others words the corners are not correctly cut.

**Comment by arkypita** [comment]
I suggest to check belt tension, mechanical backslashes, fluidity of movements, dirt on the aluminium profiles etc.

**Reply by Momboz52** [reply]
Many thanks for the suggestion.

I have model Atomstack P7 M40 40W.
I would like to check the belts tension, both the X-axis and the Y-axis ones.
For Y-axis not problem, but for the X-axis I could find any tutorial on how to do that.
Do you have any indication (or URL) on how to setup the X-axis belt tension?

My machine is new and I had the issue of loosing step from the beginning.

**Reply by Tribo-2dot0** [reply]
Ambos os eixos ajusta-se da mesma maneira, apenas solte o parafuso, estique e aperte-o novamente.

**Reply by Momboz52** [reply]
Many thanks for your help... It would help me if you could tell me which screw should I loose and tighten for the X-axis tension. This is what I couldn't manage doing.

**Reply by Tribo-2dot0** [reply]
dentro do trilho passa a correia, nos extremos das correias (e dos trilhos) verá que tem um parafuso, soltando um dos dois a correi se afrouxará, terá que puxar com os dedos até que a correia tenha uma boa tensão e aperte o parafuso novamente.
